arff.py

```diff
@@ -585,8 +607,9 @@ def _decode_attribute(self, s):
         padding, including the "\r\n" characters. 
 
         :param s: a normalized string.
-        :return: a tuple (ATTRIBUTE_NAME, TYPE_OR_VALUES).
+        :return: a tuple (ATTRIBUTE_NAME, TYPE_OR_VALUES, format).
         '''
+        format = None
```

Comment:
Could you please rename this to date_format to not shadow the built-in format and to make it more clear what this format is about?